### DISCUSSION TITLE
Table Header for Timesheet Table

### DISCUSSION THREAD

**Discussion by ofarrill** [kimai/kimai#3134] [General]
_(I am still in the transition from kimai1, so please bear with me if I haven't quite grasped the new concept yet.)_

Kimai1 had this very useful summary table on its timesheet display. Do we have something similar in kimai2?

<img width="1472" alt="Screenshot 2022-02-07 at 11 57 47" src="https://user-images.githubusercontent.com/4235974/152776352-864b5276-dfff-413b-a101-ef801c02dcfc.png">

Again, thanks for the great job on kimai generally!

**Comment by kevinpapst** [comment]
This kind of "useful" information is nice if you are using Kimai alone and your database has nothing else to do. 
If you have dozens of users / (ten-)thousands of entries, it is a database killer.
So: no, there is no replacement. You get this information from the reports mainly or by using the export preview ....

**Comment by ofarrill** [comment]
Thanks Kevin, export preview is exactly what I was looking for!
Cheers!
